src/pages/code-of-conduct.astro

```diff
@@ -1,10 +1,12 @@
 ---
 import { BaseLayout } from '~layouts';
+
+const firstHeadingId = 'code-of-conduct';
 ---
 
-<BaseLayout title="Code of Conduct">
+<BaseLayout title="Code of Conduct" skipTargetId={firstHeadingId}>
 	<div class="l-center">
-		<h2>Code of Conduct</h2>
+		<h1 id={firstHeadingId} class="h2">Code of Conduct</h1>
```

Comment:
🔍 Here, we create a `firstHeadingId` to be used both as `skipTargetId` in the layout and the ID attribute for the first heading on the page. All our other pages follow this pattern.

Reply:
Do we do this so we could theoretically have _n_ number of skip links? If we only were ever going to have a skip to main and skip to footer, couldn’t we just hardcode those two?